certmanager/generate_cert.go

```diff
@@ -83,6 +83,32 @@ const (
 // See http://www.alvestrand.no/objectid/2.5.29.17.html.
 var oidSubjectAltName = asn1.ObjectIdentifier{2, 5, 29, 17}
 
+// GenCSR generates a X.509 certificate sign request with the given options.
```

Comment:
This method returns both a CSR and the private key. Can you update the comment?

I notice we have a similar issue on line 112. I'll fix that later